### PR TITLE
Fixes #1202 & #1204: ProfileProgress - Tablet (Landscape) (Portrait) (Lowfi)  

### LINKED ISSUE #1202: ProfileProgress - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/2ff85dc5-81e1-4bc9-8c31-04e90626c9be/specs/  

Implement low-fi UI for **ProfileProgress** tablet landscape mode

**Target PR date**: 26 July 2020
**Target completion date**: 30 July 2020

### LINKED ISSUE #1204: ProfileProgress - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/3d1f3565-be7c-4458-b1e4-e68812648109  

Implement low-fi UI for **ProfileProgress** tablet portrait mode

**Target PR date**: 26 July 2020
**Target completion date**: 30 July 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1202 
- Fixes #1204 
- Implemented low-fi UI for `ProfileProgress` for tablet landscape & portrait mode
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressFragmentPresenter.kt [hunk 1]
```diff
@@ -37,14 +38,27 @@ class ProfileProgressFragmentPresenter @Inject constructor(
     // NB: Both the view model and lifecycle owner must be set in order to correctly bind LiveData elements to
     // data-bound view models.
     binding.lifecycleOwner = fragment
+
+    val spanCount = activity.resources.getInteger(R.integer.profile_progress_span_count)
+    val profileProgressLayoutManager = GridLayoutManager(activity.applicationContext, spanCount)
+    profileProgressLayoutManager.spanSizeLookup = object : GridLayoutManager.SpanSizeLookup() {
+      override fun getSpanSize(position: Int): Int {
+        return if (position == 0) {
+          /* number of spaces this item should occupy = */ spanCount
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/profileprogress/ProfileProgressFragmentPresenter.kt]: Nit:
`/* number of spaces this item should occupy= */ spanCount`

Ditto below

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt [hunk 2]
```diff
@@ -84,8 +88,14 @@ class ProfileProgressViewModel @Inject constructor(
     Transformations.map(ongoingStoryListResultLiveData, ::processOngoingStoryResult)
   }
 
-  val ongoingStoryListViewModelLiveData: LiveData<List<ProfileProgressItemViewModel>> by lazy {
-    Transformations.map(ongoingStoryListLiveData, ::processOngoingStoryList)
+  var refreshedOngoingStoryListViewModelLiveData =
+    MutableLiveData<List<ProfileProgressItemViewModel>>()
+
+  fun refresh() {
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt]: Add a kDoc mentioning why this function is needed?

Comment (rt4914) [reply app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt]: handleOnConfigurationChange()

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested nits. LGTM

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressFragmentPresenter.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/profileprogress/ProfileProgressFragmentPresenter.kt]: Done

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**